content/javascript/concepts/jquery/jquery.md

```diff
@@ -0,0 +1,60 @@
+---
+Title: 'jQuery'
+Description: 'jQuery is a popular JavaScript library intended to simplify JavaScript programming.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'DOM'
+  - 'Events'
+  - 'jQuery'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+**jQuery** is a JavaScript library that simplifies common tasks that once required many lines of code (e.g., [AJAX](https://www.codecademy.com/resources/docs/javascript/ajax), DOM manipulation, and [event handling](https://www.codecademy.com/resources/docs/javascript/events)), and wraps them inside a single method call.
+
+## Usage
+
+jQuery can be added to a website by either downloading the library from [jQuery.com](https://jquery.com/download/), or including it from a [CDN](https://www.codecademy.com/resources/docs/general/cdn).
+
+Including a downloaded library looks like this (downloaded into the same directory as the page using it):
+
+```html
+<head>
+  <script src="jquery-3.6.0.min.js"></script>
+</head>
+```
+
+Including from a CDN (in this case Google) looks like this:
+
+```html
+<head>
+  <script src="jquery-3.6.0.min.js"></script>
```

Comment:
Is the src here correct? It appears to be the same as the downloaded library example.